Airports/EGPF/SMR/Labels.txt

```diff
@@ -3,7 +3,6 @@
 "A2" N055.52.38.524 W004.25.08.597 standHold
 "A3" N055.52.16.179 W004.25.38.519 standHold
 "B1" N055.52.36.703 W004.25.16.450 standHold
-"C1" N055.52.20.502 W004.25.47.990 standHold
```

Comment:
Same here - just comment this out with a ; at the start of the line, rather than deleting it